src/styles/rootlayout.css

```
@@ -0,0 +1,40 @@
+/*
+    전체 width 설정
+*/
+
+/* 기본: 모바일 */
+.app {
+  position: relative;
+  max-width: 100%;
+  min-height: 100vh;
+  min-height: 100dvh;
+  margin: 0 auto;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)